rcl/include/rcl/context.h

```diff
@@ -240,6 +240,31 @@ RCL_WARN_UNUSED
 rcl_context_instance_id_t
 rcl_context_get_instance_id(rcl_context_t * context);
 
+/// Returns the context domain id.
+/**
+ * The given context must be non-`NULL` and be valid.
+ * If context is `NULL`, then `0` will be returned.
+ * If context is uninitialized, then it is undefined behavior.
```

Comment:
@ivanpauno I think this is covered by the first sentence.

Reply:
See https://github.com/ros2/rcl/pull/718/commits/85708bed10368e0ddc973c1c44fce2b8d7f513a7.